src/core_modules/capture-core/flow/typeDeclarations.js

```diff
@@ -1,7 +1,6 @@
 // @flow
 /* eslint-disable */
 import type { Store } from 'redux';
-import { methods } from '../src/trackerOffline/trackerOfflineConfig.const';
```

Comment:
yeah you right this is used in this file but the path `../src/trackerOffline/trackerOfflineConfig.const` is wrong at the moment. Fixing the path instead of deleting it ;)